src/components/dialog/dialog.component.ts

```diff
@@ -103,12 +103,6 @@ export default class SlDialog extends ShoelaceElement {
    */
   @property({ reflect: true }) label = '';
 
-  /**
```

Comment:
Why is this being removed? And does this change have anything to do with the spinner? If not, we should probably leave it out of this PR.

Reply:
It's a mistake I made from a previous PR (the one for the recent huge update). The property doesn't actually exist -- you'll see that the description is identical to another property that does exist, because I'd copy-pasta-ed it originally. When I updated the Dialog component before, I was going to add a new property for the header icon & created that line, but later realized that it made more sense as a slot and didn't need to be a new property. But I forgot to delete this line that I'd added. I only saw it later & realized that I forgot to delete it! It's such a small change that I thought it would be okay to add here, but am happy to move this bit to another PR if you prefer.